lagunita/lms/templates/static_templates/copyright.html

```diff
@@ -9,17 +9,20 @@
 <%namespace name='static' file='/static_content.html'/>
 <%namespace name='theme_settings' file='/theme-settings.html'/>
 
-<%block name="pagetitle">${_('Copyright Complaints')}</%block>
-
 <%
   theme_vars = theme_settings.get_theme_vars()
   effective_date = theme_vars['effective_date']
   contact_email = '<a href="mailto:{email}">{email}</a>'.format(email=settings.CONTACT_EMAIL)
+  tos_links = theme_settings.get_tos_links()
 %>
 
+<%block name="pagetitle">${_('Copyright Complaints')}</%block>
+
 <main id="main" aria-label="Content" tabindex="-1">
     <section class="container copyright">
-        <h2 id="copyright">${_('Copyright Complaints')}</h2>
+        <%include file="menu_tos.html" args="current_page='copyright'" />
```

Comment:
Change name to remove `tos`;
maybe call it, `menu_legal_links.html` or some such.